src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  static removeIfDead(animal) {
+    // write code
+    if (animal.health <= 0) {
+      Animal.alive = Animal.alive.filter(
+        (aliveAnimal) => aliveAnimal !== animal,
+      );
+    }
+  }
```

Comment:
[CODE PERFORMANCE] - The `removeIfDead` method is currently using a conditional check before filtering. This check is unnecessary because the `filter` method can handle the condition internally. You can directly filter `Animal.alive` without the `if` statement, improving the performance by avoiding the unnecessary check.